app/locales/ms/intro.json

```diff
@@ -0,0 +1,14 @@
+{
+    "private_kit":"Private Kit",
+    "intro1_para1":"Direkakan dengan eselamatan data peribadi anda dan dengan tujuan untuk melindungi privasi, MIT Private Kit ialah sesuatu app generasi baru untuk melogkan lokasi dengan selamat.",
+    "next":"LAMAN SETERUSNYA",
+    "back":"LAMAN LEPAS",
+    "start":"MULA",
+    "intro2_title1":"Less than 100KB",
+    "intro2_para1":"Private Kit’s trail generator logs your device’s location data in under 100KB of space – less space than a single picture.",
+    "intro2_title2":"You are in charge",
+    "intro2_para2": "Data Never Leaves Your Device Without Your Consent",
+    "intro3_title1":"The Future",
+    "intro3_para1": "The Next Step in Solving Today's and Tomorrow’s Problems Enabling individuals to log their location trail offers new opportunities for researchers studying pandemic tracking, refugee migration, and community traffic analysis.",
```

Comment:
please translate these keys too

Reply:
I left these in English because it is grammatically erratic to write in Malay in this way. I will look into it further soon.

Reply:
eda4ca0